src/dune/utop.mli

```diff
@@ -8,4 +8,6 @@ val utop_exe : string
 
 val is_utop_dir : Path.Build.t -> bool
 
+val libs_under_dir : Super_context.t -> db:Lib.DB.t -> dir:Import.Path.t -> Lib.L.t
```

Comment:
Let's not use `Import` in the mli. It should just be `Path.t` and `Stdune` be opened.